scripts/dappnode_install.sh

```diff
@@ -315,9 +372,12 @@ if [ "$ARCH" == "amd64" ]; then
     installSgx
 
     echo -e "\e[32mInstalling extra packages...\e[0m" 2>&1 | tee -a $LOGFILE
-    installExtraDpkg
+    installExtraDpkg # TODO: Why is this being called twice?
 fi
 
+echo -e "\e[32mAdding user to docker group...\e[0m" 2>&1 | tee -a $LOGFILE
+addUserToDockerGroup
```

Comment:
if all distros (debian, ubuntu..) call `addUserToDockerGroup()` here, we can remove it from `dappnode_install_pre.sh` right?